src/components/search/tests/FacetListBase.test.jsx

```diff
@@ -3,16 +3,18 @@ import { act, screen, fireEvent } from '@testing-library/react';
 import '@testing-library/jest-dom/extend-expect';
 import { FREE_ALL_TITLE } from '../SearchFilters';
 
-import FacetListFreeAll from '../FacetListFreeAll';
+import FacetListBase from '../FacetListBase';
 import { FACET_ATTRIBUTES, SUBJECTS } from '../data/tests/constants';
 import { renderWithRouter } from '../../../utils/tests';
-import { NO_OPTIONS_FOUND } from '../data/constants';
+import { NO_OPTIONS_FOUND, SHOW_ALL_NAME } from '../data/constants';
+import SearchData from '../SearchContext';
 
 const propsForNoItems = {
   items: [],
   title: FREE_ALL_TITLE,
-  showAllCatalogs: false,
-  setShowAllCatalogs: () => {},
+  attribute: SHOW_ALL_NAME,
+  isBold: true,
+  facetValueType: 'bool',
```

Comment:
It'd be great if it wasn't necessary to use `facetValueType`, but I don't have an alternative approach to suggest right now haha. That said, I think it's fine for the time being.

Reply:
Yeah, this was why they originally were different components, because we deal with what the value is pretty differently based on what type they are. I am definitely not in love with this solution; maybe we'll think of something better next time we're here. 😄